pkg/graph/resolvers/plan_discussion.go

```diff
@@ -42,11 +44,27 @@ func CreatePlanDiscussion(
 		return nil, err
 	}
 
-	result, err := store.PlanDiscussionCreate(logger, planDiscussion)
+	err = UpdateTaggedHTMLMentionsAndRawContent(ctx, store, &planDiscussion.Content, getAccountInformation)
+
+	if err != nil {
+
+		logger.Info("not all mentions were able to be updated")
+		//TODO: do we need to stop execution here? Should we silently continue instead? Should we filter out any bad tags?
+		// return nil, err
+	}
+
+	discussion, err := store.PlanDiscussionCreate(logger, planDiscussion)
 	if err != nil {
 		return nil, err
 	}
 
+	//TODO: should we put this in a transaction? Likely, discussions should be saved even if tags arent'
```

Comment:
I'd like some feedback on if you think this should be blocking or not. I felt it was better to write the discussion than not. Also, what do you think about continuing if we can't match a tag to a database entity up above?

Reply:
I think I'd rather the functionality of tagging be atomic. I think if we can't match up a tag to an entity in the DB (currently an account or a soln), that somewhat invalidates the content of the tag, and suceeding (silently) might lead to unexpected behavior

It's a bit more complicated in the case of creating the tag, I think -- since the plan discussion has to be created before the tag is created (so it has reference to the ID)

This is probably worth syncing on to see if we can come up with a way (maybe just for these 2 DB calls) to wrap in a transaction.

Reply:
That sounds good! I have a few ideas for this, I think perhaps a similar approach to how we wrap delete statements in a transaction would also be good!